README.md

```diff
@@ -10,79 +10,56 @@ Frontend documentation see \kot\frontend\README.md.
 
 ## Hosting Options
 
-* Production: React website hosted via Django server
-* Development: run React ('npm start') and Python ("python manage.py runserver") seperately. Allows for React and python to auto watch and reload.
+- Production: React website hosted via Django server
+- Development: run React ('npm start') and Python ("python manage.py runserver") seperately. Allows for React and python to auto watch and reload.
 
 ## Site Map
 
-* <http://localhost:8000/>
-* <http://localhost:8000/api/v1/dice/>
-* <http://localhost:8000/admin/>
-* <http://localhost:8000/users/>
+- <http://localhost:8000/>
+- <http://localhost:8000/api/v1/dice/>
```

Comment:
This should really be removed. This isn't how the game will use dice and was something that you added as a separate thing to test sending and receiving messages on the front end.

Reply:
Rolling dice is a stateless operation, so one off endpoints calls seems fine. Once we have a command for sending dice (which would be a dup of effort) we can disable this I guess.